.github/workflows/ci-e2e-cassandra.yml

```diff
@@ -48,6 +48,14 @@ jobs:
       run: bash scripts/cassandra-integration-test.sh ${{ matrix.version.major }} ${{ matrix.version.schema }} ${{ matrix.jaeger-version }}
       env:
         SKIP_APPLY_SCHEMA: ${{ matrix.create-schema == 'auto' && true || false }}
+     
+    - name: Upload Metrics Artifact
+      uses: actions/upload-artifact@v4
+      if: ${{ matrix.jaeger-version }} == "v2"
+      with:
+              name: cassandra-metrics-${{ matrix.version }}
+              path: ./metrics/cassandra_metrics.txt
+              retention-days: 60
```

Comment:
60 days could force us to exceed storage quota from GitHub. We only need 60 days for uploads from tagged release, all other uploads can have 7 days retention.

Reply:
how do we know if its a tagged release run or a normal run ?